frontend/rsbuild.config.ts

```diff
@@ -73,6 +74,21 @@ export default defineConfig({
         new NodePolyfillPlugin({
           additionalAliases: ['process'],
         }),
+        new MonacoWebpackPlugin({
+          languages: ['yaml'],
```

Comment:
```suggestion
          languages: ['yaml', 'typescript', 'javascript', 'json'], 
```
maybe you need to set more languages

Reply:
I don't think so, because those languages are part of monaco itself

Reply:
oh ok interesting, You know best, I thought the Webpack plugin wouldn't add these workers if you don't set them explicitly